draft-ietf-netconf-over-tls13.md

```diff
@@ -85,8 +89,19 @@ data.
 
 # Cipher Suites
 
-Implementations that support TLS 1.3 {{I-D.ietf-tls-rfc8446bis}} are REQUIRED
-to support the mandatory-to-implement cipher suites listed in
+Implementations MUST support TLS 1.2 {{RFC5246}} and are REQUIRED to
+support the mandatory-to-implement cipher suite.
+
+Implementations MAY implement additional TLS cipher suites that provide
+mutual authentication {{RFC5246}} and confidentiality as required by
+NETCONF {{RFC6241}}.
+
+Implementations SHOULD support TLS 1.3 {{I-D.ietf-tls-rfc8446bis}} and,
+if implemented, MUST prefer to negotiate TLS 1.3 over earlier versions
+of TLS.
```

Comment:
```suggestion
Implementations that support more than one version of TLS MUST
prefer later versions of TLS over earlier versions of TLS.
```

Reply:
Russ suggested this version of the text to better future proof it.

Reply:
Decided to leave it alone. When there's a new TLS version there will no doubt be a new cipher suite so this section will need to change regardless.